noxconfig.py

```diff
@@ -20,7 +20,7 @@ class Config:
         "resources",
         "config_data_model.py",
     )
-    python_versions = ["3.9", "3.10", "3.11", "3.12", "3.13"]
+    python_versions = ["3.10", "3.11", "3.12", "3.13"]
 
 
 PROJECT_CONFIG = Config()
```

Comment:
lgtm you could also do it at a later point, but could switch to the BaseConfig:
https://github.com/exasol/python-toolbox/releases/tag/1.10.0

Hopefully by next year, we'll have something there to skip not required steps in certain projects...like building docs.